doc/source/reference/maskedarray.generic.rst

```diff
@@ -492,8 +487,10 @@ the last two where the inputs were masked.
 Ignoring extreme values
 -----------------------
 
-Let's consider an array ``d`` of random floats between 0 and 1. We wish to
+Let's consider an array ``d`` of floats between 0 and 1. We wish to
 compute the average of the values of ``d`` while ignoring any data outside
-the range ``[0.1, 0.9]``::
+the range ``[0.2, 0.9]``::
 
-   >>> print ma.masked_outside(d, 0.1, 0.9).mean()
+   >>> d = np.linspace(0, 1, 20)
```

Comment:
@seberg - how does this look now?

Reply:
Works for me. First thought you were talking about the rendered docs, (which CircleCI builds as artifacts, although it will need a few minutes).